admin-dev/themes/new-theme/js/pages/product/product-map.ts

```diff
@@ -162,6 +162,7 @@ export default {
     defaultDescription: '.serp-default-description',
     watchedDescription: '.serp-watched-description',
     watchedMetaUrl: '.serp-watched-url:input',
+    tagFields: 'input.js-taggable-field',
```

Comment:
```suggestion
    tagFields: '.js-taggable-field',
```

At the same time than fixing the other comment, remove the input element selector, so in the future in case it's not an input, it won't break, the JS class seems enough

Reply:
Ok, not related to the PR, but added a todo